PyInstaller/depend/utils.py

```diff
@@ -79,9 +79,11 @@ def _write_long(f, x):
                         # Name inside a zip archive.
                         # TODO use .pyo suffix if optimize flag is enabled.
                         if type(mod) is modulegraph.Package:
-                            new_name = mod.identifier.replace('.', os.sep) + os.sep + '__init__' + '.pyc'
+                            new_name = mod.identifier.replace(
+                                '.', os.sep) + os.sep + '__init__' + '.pyc'
                         else:
-                            new_name = mod.identifier.replace('.', os.sep) + '.pyc'
+                            new_name = mod.identifier.replace(
+                                '.', os.sep) + '.pyc'
```

Comment:
Please remove all white-space and reformatting changes, here and in the other files of this commit. Thanks.